storage/postgres/postgres.go

```diff
@@ -97,6 +85,26 @@ func NewPostgresDatastore(uri string, opts ...PostgresOption) (*Postgres, error)
 		p.maxTypesInTypeDefinition = defaultMaxTypesInDefinition
 	}
 
+	policy := backoff.NewExponentialBackOff()
+	policy.MaxElapsedTime = 1 * time.Minute
+	var pool *pgxpool.Pool
+	attempt := 1
+	err := backoff.Retry(func() error {
+		var err error
+		pool, err = pgxpool.Connect(context.Background(), uri)
+		if err != nil {
+			p.logger.Info(fmt.Sprintf("failed to initialize Postgres connection: (attempt #%d)", attempt))
```

Comment:
Use fields to indicate count. Keep the message brief as it was.

Reply:
👍 Thanks! Updated.